README.md

```diff
@@ -277,7 +277,7 @@ Writing a new type definition is as simple as creating a new package. Start by c
 
 * **main** The entry point to the definition (canonical to "main" in NPM's `package.json`)
 * **browser** A string or map of paths to override when resolving (following the [browser field specification](https://github.com/defunctzombie/package-browser-field-spec))
-* **ambient** Denote that this definition _must_ be installed as ambient
+* **ambient** Denote that this definition should be installed as ambient to the most likely target environment
```

Comment:
Was trying to be explicit with the concept of it as a suggestion to the user of the typing by reflecting some of the language of the section that follows about ambient definitions. Hopefully by pointing out a reason the suggestion might be ignored, such as trying to use a type definition outside of the platform the module author and/or typing author expected. "target environment" is symmetrical with the section explaining ambient definitions as opposed to "target platform" which might read clearer out of context.